storagedriver/azure/blockblob.go

```diff
@@ -0,0 +1,24 @@
+package azure
+
+import (
+	"fmt"
+	"io"
+
+	azure "github.com/MSOpenTech/azure-sdk-for-go/clients/storage"
+)
+
+// azureBlockStorage is adaptor between azure.BlobStorageClient and
+// blockStorage interface.
+type azureBlockStorage struct {
+	azure.BlobStorageClient
+}
+
+func (b *azureBlockStorage) GetBlobRange(container, blob string, start, length int64) (io.ReadCloser, error) {
```

Comment:
Sorry if that was confusing but I meant that this should be called `GetSectionReader`. It doesn't have to implement seek or use `io.SectionReader` but that defines what is happening.